staging/cert-manager-setup/templates/install-crds.yaml

```diff
@@ -0,0 +1,86 @@
+{{- if .Values.installCertManager }}
+{{ if not (.Capabilities.APIVersions.Has "certmanager.k8s.io/v1alpha1") }}
+---
+{{.Files.Get "files/crds.yaml"}}
+{{ end }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: cert-manager-setup-crds
+  annotations:
+    "helm.sh/hook": pre-install, pre-upgrade
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": hook-succeeded
+data:
+  crds.yaml: |-
+{{.Files.Get "files/crds.yaml" | printf "%s" | indent 4}}
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: cert-manager-setup-crds
+  annotations:
+    "helm.sh/hook": pre-install, pre-upgrade
+    "helm.sh/hook-weight": "-4"
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    spec:
+      serviceAccountName: cert-manager-setup-crds
+      containers:
+        - name: cert-manager-setup-crds
+          image: "bitnami/kubectl:1.16.2"
+          volumeMounts:
+            - name: cert-manager-setup-crds
+              mountPath: /etc/cert-manager-setup-crds
+              readOnly: true
+          command: ["kubectl", "apply", "-f", "/etc/cert-manager-setup-crds"]
+      volumes:
+        - name: cert-manager-setup-crds
+          configMap:
+            name: cert-manager-setup-crds
+      restartPolicy: OnFailure
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: cert-manager-setup-crds
+  annotations:
+    "helm.sh/hook": pre-install, pre-upgrade
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": hook-succeeded
```

Comment:
Can you also add "before-hook-creation" just to be safe?

"helm.sh/hook-delete-policy": hook-succeeded, before-hook-creation

Reply:
Please apply this also to other places (even in kommander chart)

Reply:
done for this chart, there's many places where this needs to be applied. will do it in another PR

Reply:
other PR here: https://github.com/mesosphere/charts/pull/242